IBPSA/package.mo

```diff
@@ -486,7 +486,7 @@ annotation (
 version="3.0.0",
 versionDate="2018-09-28",
 dateModified = "2018-09-28",
-uses(Modelica(version="4.0.0")),
+uses(Modelica(version="4.0.0"), Buildings(version="8.0.0")),
```

Comment:
I think the _Buildings_-Library should not be used by the _IBPSA_. Normally it is vice versa. I cannot find any model implemented by this PR that uses the _Buildings_.

Reply:
Not sure what to do here. Should I just remove that String from the code?

Reply:
I would suggest to try and remove "Buildings(version="8.0.0")" and check the IBPSA package. If no error occurs, the library isn't used in any model of the IBPSA and all is fine without the additional string. :)